std/stdio.d

```diff
@@ -903,6 +903,56 @@ Throws: $(D ErrnoException) if the file is not opened or if the call to $(D fwri
     }
 
 /**
+Sets the $(LREF File) to "raw" mode, such that all subsequent reads and writes will
+behave like $(LREF rawRead) and $(LREF rawWrite).
+
+Calls $(LREF flush) (all platforms), and sets the $(D FILE*) to binary mode on Windows.
+
+This can be used to set $(D stdin) and $(D stdout), which are implicitly opened in text
+mode, to binary mode on Windows.
+
+See_Also:
+$(LREF rawRead)
+$(LREF rawWrite)
+ */
+    void setRawMode()
```

Comment:
The terminology is binary vs. text. Then we probably need a means to get the current mode, so there's no escaping the API:

```d
enum FileMode { text, binary }
void getMode(FileMode);
void setMode(FileMode);
```